test/normalizer/nullableAddsNullToType.json

```diff
@@ -0,0 +1,139 @@
+{
+  "name": "Nullable adds null to type",
+  "in": {
+    "$id": "a",
+    "type": "object",
+    "properties": {
+      "a": {
+        "const": "",
+        "nullable": true
+      },
+      "b": {
+        "const": null,
+        "nullable": true
+      },
+      "c": {
+        "enum": ["a", "b"],
+        "nullable": true
+      },
+      "d": {
+        "enum": ["", null],
+        "nullable": true
+      },
+      "e": {
+        "type": "string",
+        "nullable": true
+      },
+      "f": {
+        "type": "null",
+        "nullable": true
+      },
+      "g": {
+        "type": "string",
+        "const": "",
+        "nullable": true
+      },
+      "h": {
+        "type": "string",
+        "const": null,
+        "nullable": true
+      },
+      "i": {
+        "type": "string",
+        "enum": ["a", "b"],
+        "nullable": true
+      },
+      "j": {
+        "type": "string",
+        "enum": ["", null],
+        "nullable": true
+      },
+      "k": {
+        "type": ["string", "integer"],
+        "nullable": true
+      },
+      "l": {
+        "type": ["string", "null"],
+        "nullable": true
+      },
+      "m": {
+        "anyOf": [{"type": "string"}, {"type": "integer"}],
+        "nullable": true
+      },
+      "n": {
+        "anyOf": [{"type": "string"}, {"type": "null"}],
+        "nullable": true
+      },
+      "o": {
+        "oneOf": [{"type": "string"}, {"type": "integer"}],
+        "nullable": true
+      },
+      "p": {
+        "oneOf": [{"type": "string"}, {"type": "null"}],
+        "nullable": true
+      }
+    },
+    "required": [],
+    "additionalProperties": false
+  },
+  "out": {
+    "$id": "a",
+    "type": "object",
+    "properties": {
+      "a": {
+        "enum": ["", null]
+      },
+      "b": {
+        "enum": [null]
+      },
+      "c": {
+        "enum": ["a", "b", null]
+      },
+      "d": {
+        "enum": ["", null]
+      },
+      "e": {
+        "type": ["string", "null"]
+      },
+      "f": {
+        "type": ["null", "null"]
+      },
```

Comment:
@mdmower-csnw: The output here and cases `l`, `n`, `p` don't seem ideal with their redundant `null`. I suppose it doesn't matter since the types are generated correctly.

Reply:
Deduplication was trivial so opted to handle it in [a5db6da](https://github.com/CSNW/json-schema-to-typescript/pull/1/commits/a5db6da52b5992917f08f911994d8ed2df58f1b5).